src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -2514,5 +2514,53 @@ public SafeHandle NativeHandle
                 return new NUI.SafeNativeWindowHandle(this);
             }
         }
+
+        /// <summary>
+        /// It is used to make a given window a modal one. 
+        /// Once it's set as a modal window, only that window can receive user input.
+        /// </summary>
+        /// <param name="modal"> If modal is true, set the modal window or unset.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetModal(bool modal)
+        {
+            Interop.Window.SetModal(SwigCPtr, modal);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Gets whether the window is modal or not.
+        /// </summary>
+        /// <returns>Returns true if the full screen sized window is.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsModal()
+        {
+            bool ret = Interop.Window.IsModal(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
```

Comment:
What about to provide a single property instead of having separate setter and getter methods like below?

```suggestion

        /// <summary>
        /// Gets or sets a value indicating whether the window is modal or not.
        /// </summary>
        [EditorBrowsable(EditorBrowsableState.Never)]
        public bool IsModal
        {
            get
            {
                bool ret = Interop.Window.IsModal(SwigCPtr);
                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
                return ret;
            }
            set
            {
                Interop.Window.SetModal(SwigCPtr, modal);
                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
            }
        }
```

Reply:
Reference of IsModal

https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.windowing.overlappedpresenter.ismodal?view=windows-app-sdk-1.5

Reply:
Thank you for your review. I will change this PR with a single property. :)